tests/v2/validation/clustertemplates/provisioning/README.md

```diff
@@ -0,0 +1,72 @@
+## Cluster template Provisioning Config
+
+For your config, you will need everything in the [Prerequisites](../README.md) section on the previous readme along with and at least one [Cloud Credential](#cloud-credentials). 
```

Comment:
can you explicitly say somewhere in the description this is for provisioned / node driver clusters only?